src/main/java/frc/robot/hardware/motor/sparkmax/SparkMaxWrapper.java

```diff
@@ -13,4 +15,8 @@ public double getVoltage() {
 		return getAppliedOutput() * getBusVoltage();
 	}
 
+	public Rotation2d getVelocityPerSecond() {
```

Comment:
Meters per second? Rotations per second? Sandwiches per second?

Reply:
Revolutions is more standard as a name for continuous rotation. See the other conversation on this function